pywt/_multidim.py

```diff
@@ -276,7 +277,7 @@ def idwtn(coeffs, wavelet, mode='symmetric', axes=None):
     return coeffs['']
 
 
-def swt2(data, wavelet, level, start_level=0):
+def swt2(data, wavelet, level, start_level=0, axes=(-2, -1)):
```

Comment:
Okay, different return formats plus consistency with ``dwt2``/``dwtn`` seems like a good enough reason to add ``axes``.